src/systems/systems/src/apu/mod.rs

```diff
@@ -1459,7 +1459,7 @@ pub mod tests {
             let mut test_bed = test_bed_with()
                 .starting_apu()
                 .and()
-                .indicated_altitude(Length::new::<foot>(25000.))
+                .indicated_altitude(Length::new::<foot>(25000.01))
```

Comment:
I guess this requires a bit of explanation. I understand it has something with the UOM upgrade, but shouldn't we rather update the assertion below or even better the code behind this? I think reading the name of the test, we have failed the intent of this test now.

At the very least it deserves a comment.

Reply:
Yeah I failed to investigate the exact cause, but I'd suspect something like added precision on the value of a foot in new uom package. The test being "at or above", it didn't felt as killing the test purpose though.
If my rust analyzer stops failing I might have a chance at finding the root cause....